app/src/models/github-repository.ts

```diff
@@ -58,13 +58,9 @@ export class GitHubRepository {
    * Objects with the same hash are guaranteed to be structurally equal.
    */
   public get hash(): string {
-    return `${this.dbID}+
-      ${this.defaultBranch}+
-      ${this.private}+
-      ${this.cloneURL}+
-      ${this.name}+
-      ${this.htmlURL}+
-      ${this.owner.hash}+
-      ${this.parent && this.parent.hash}`
+    return `${this.dbID}+${this.defaultBranch}+${this.private}+${
```

Comment:
To be honest, I avoided changing this implementation because I just wanted the strings to be more readable. Can we move discussion about changing the implementation to a `tech-debt` issue and work on that later?

Reply:
https://github.com/desktop/desktop/issues/5988